schema/src/bindings/typescript.ts

```diff
@@ -89,12 +85,24 @@ export const typesInterface = (schemas: JsonSchema[]): string => {
  * Convert the `title` of a JSON Schema to the name of a Typescript
  * type, including the interfaces and unions generated below.
  */
-export const titleToType = (title: string): string => {
+export const titleToType = (title: string | number): string => {
```

Comment:
Why does the `| number` need to be added here? Is the `title` of a JSON Schema object allowed to be a number.

Reply:
Seems to be leftover from a previous ObjectType typing, I've now removed it.